python_modules/libraries/dagster-databricks/dagster_databricks/ext.py

```
@@ -0,0 +1,165 @@
+                    raise DagsterExternalExecutionError(
+                        f"Error running Databricks job: {run.state.state_message}"
+                    )
+                time.sleep(5)
+
+
+ExtDatabricks = ResourceParam[_ExtDatabricks]
+
+_CONTEXT_FILENAME = "context.json"
+
+
+@contextmanager
+def dbfs_tempdir(dbfs_client: files.DbfsAPI) -> Iterator[str]:
+    dirname = "".join(random.choices(string.ascii_letters, k=30))
+    tempdir = f"/tmp/{dirname}"
+    dbfs_client.mkdirs(tempdir)
+    try:
+        yield tempdir
+    finally:
+        dbfs_client.delete(tempdir, recursive=True)
+
+
+class ExtDbfsContextInjector(ExtContextInjector):
+    def __init__(self, *, client: WorkspaceClient):
+        super().__init__()
+        self.dbfs_client = files.DbfsAPI(client.api_client)
+
+    @contextmanager
+    def inject_context(self, context: "ExtContextData") -> Iterator[ExtParams]:
+        with dbfs_tempdir(self.dbfs_client) as tempdir:
+            path = os.path.join(tempdir, _CONTEXT_FILENAME)
+            contents = base64.b64encode(json.dumps(context).encode("utf-8")).decode("utf-8")
+            self.dbfs_client.put(path, contents=contents, overwrite=True)
+            yield {"path": path}
+
+
+class ExtDbfsMessageReader(ExtBlobStoreMessageReader):
+    def __init__(self, *, interval: int = 10, client: WorkspaceClient):
+        super().__init__(interval=interval)
+        self.dbfs_client = files.DbfsAPI(client.api_client)
+
+    @contextmanager
+    def setup(self) -> Iterator[ExtParams]:
+        with dbfs_tempdir(self.dbfs_client) as tempdir:
+            yield {"path": tempdir}
+
+    def download_messages_chunk(self, index: int, params: ExtParams) -> Optional[str]:
+        message_path = os.path.join(params["path"], f"{index}.json")
+        try:
+            raw_message = self.dbfs_client.read(message_path)
+            return raw_message.data
+        except IOError:
+            return None
```

just silently swallow the error?

At minimum we should warn or something.